frontend/server/src/DAO/CourseIdentityRequest.php

```diff
@@ -12,4 +12,89 @@
  * @access public
  */
 class CourseIdentityRequest extends \OmegaUp\DAO\Base\CourseIdentityRequest {
+    /**
+     * @return list<array{accepted: bool|null, admin?: array{name: null|string, username: string}, country: null|string, country_id: null|string, last_update: null|string, request_time: string, username: string}>
```

Comment:
falta propagar los timestamps acá

```suggestion
     * @return list<array{accepted: bool|null, admin?: array{name: null|string, username: string}, country: null|string, country_id: null|string, last_update: \OmegaUp\Timestamp|null, request_time: \OmegaUp\Timestamp, username: string}>
```

y en todos los demás lugares donde esto se llama y así sucesivamente.

Reply:
Listo